src/utils/jira/getMostRelevantJiraTicket.ts

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+import { backendURL } from "../../constants";
+
+export default async function getMostRelevantJiraTicket({
+  userEmail,
+  prTitle,
+}: {
+  userEmail: string;
+  prTitle: string;
```

Comment:
This depends on github?

Reply:
What if the user does not have github?

Reply:
If the user doesn't have GitHub, he wouldn't be able to get to try our product for free in the first place. So yes, this depends on GitHub. 

As we expand to the counterparts (Bitbucket, Notion, etc.) we can build some sort of middle layer that allows to connect any git system with any ticketing system. For the moment, let's go deep on the VS Code + GitHub + Jira integration.